//----------------------------------------------------------------------------
// Copyright 2023, Ed Keenan, all rights reserved.
//----------------------------------------------------------------------------

syntax = "proto3";

option optimize_for = LITE_RUNTIME;

message JointData_proto
{
	optional uint32 numJoints = 1;
	repeated uint32    poData = 2;
}

// -- End of File ---
